ui/components/component-library/banner-tip/banner-tip.stories.tsx

```diff
@@ -147,7 +111,7 @@ ActionButton.args = {
     'Use actionButtonLabel for action text, actionButtonOnClick for the onClick handler, and actionButtonProps to pass any ButtonLink prop types such as iconName',
 };
```

Comment:
apart from this console error in story everything works fine. 
![image](https://github.com/MetaMask/metamask-extension/assets/79097544/13971c01-84db-44ae-ac8e-189329223f17)

tried type assertion but it didnt work either. 
```
  actionButtonProps: {
    iconName: IconName.Arrow2Right,
    iconPositionRight: true,
  } as ButtonLinkProps<'button'>,
```

Reply:
Ah this looks like a mistake it should be `endIconName`

Reply:
done [`here`](https://github.com/MetaMask/metamask-extension/pull/20538/commits/574ec7f09f8c59f7d89da837f39ff9e1fe5a4611)